Create ui-c8y-1021-2-1-fix-add-type-validator-to-the-alarm-event-selector-7423-graft.md (#2627)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-2-1-fix-add-type-validator-to-the-alarm-event-selector-7423-graft.md

---
date: ""

> title: fix: add type validator to the alarm/event selector (#7423) [GRAFT][release/cd] (#7558)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Alarm and event selector input validation added
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61126
version: 1021.2.1
---

> fix: add type validator to the alarm/event selector (#7423) [GRAFT][release/cd] (#7558)

Author: The Claude AI suggested to use the following body:
```suggestion
The alarm and event selector in the Cumulocity IoT user interface previously allowed invalid input to be entered. With this fix, input validation has been added to the alarm and event selector to ensure only valid input is accepted. This change improves the user experience and prevents potential errors or unexpected behavior when using the alarm and event selector.
```

> The alarm and event selector in the Cumulocity IoT user interface previously allowed invalid input to be entered. With this fix, input validation has been added to the alarm and event selector to ensure only valid input is accepted. This change improves the user experience and prevents potential errors or unexpected behavior when using the alarm and event selector.

Reviewer: ```suggestion
The alarm and event selector in the {{< product-c8y-iot >}} UI previously allowed to enter invalid input. With this fix, input validation has been added to the alarm and event selector to ensure that only valid input is accepted. This change improves the user experience and prevents potential errors or unexpected behavior when using the alarm and event selector.
```
